will_legislate_for_money/urls.py

```diff
@@ -26,4 +28,13 @@
     url(r'^recent-bills/$', public_official_views.bills_index, name="bills_index"),
     url(r'^legislators/', include('public_officials.urls')),
     url(r'^admin/', admin.site.urls),
+    url(r'^logout/$', public_official_views.log_out, name="log_out"),
+    url(r'^oauth/', include('social_django.urls', namespace='social')),
+    url(r'^accounts/profile/', public_official_views.user_show, name="user_show"),
+    url(r'^accounts/login/', auth_views.login, name='login'),
```

Comment:
urls look solid all around, great job incorporating an external application -- not an easy task 👍